### ISSUE TITLE
remove the sharebuttons 

### ISSUE THREAD

**Issue by anmue** [electerious/Lychee#570]
Hi,
is there a sulution to remove the sharebuttons? 

best regards
andy

**Comment by electerious** [comment]
I'm sure there's a way to do this, but I can't provide any help for customization.

Editing [this function](https://github.com/electerious/Lychee/blob/master/src/scripts/header.js#L111) could be a good starting point. Maybe that helps a bit.

**Comment by albino-gryphon** [comment]
I hope that this suggestion is acceptable to post here.  First time commenting on anything in Github.

I was able to achieve this by editing the main.js file for my purposes.  I wanted the share button removed when not logged in for both albums and pictures.  I altered the following section:

$("#button_share, #button_share_album").removeClass("button--eye").addClass("button--share").find("use").attr("xlink:href","#share")

Changing it to this:

$("#button_share, #button_share_album").remove()
